.github/workflows/publish-types.yaml

```diff
@@ -18,6 +26,10 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '20'
+          registry-url : 'https://registry.npmjs.org'
+
+      - name: Configure npm
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
```

Comment:
Should we add this to https://github.com/dev-launchers/strapiv4/blob/main/.npmrc directly?

Reply:
yes, we should, adding this line

      - name: Configure npm
        run: |
          echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
          echo "registry=https://registry.npmjs.org" >> ~/.npmrc